cliquet/resource.py

```diff
@@ -198,7 +192,14 @@ def register_service(endpoint_type):
         name = viewset.get_service_name(endpoint_type, resource)
 
         service = Service(name, path, depth=depth,
-                          **viewset.service_arguments or {})
+                          **viewset.get_service_arguments())
+
+        # Attach viewset and resource to the service for later reference.
+        service.viewset = viewset
+        service.resource = resource
```

Comment:
Oh, this is a great helper :)

Just wondering (issue for later if relevant) : shouldn't we add a reified property on request instead ? (using a closure for example)